.github/workflows/helm-weekly-release-pr.yaml

```diff
@@ -0,0 +1,30 @@
+name: helm-weekly-release-pr
+
+on:
+  schedule:
+    - cron: '0 12 * * 1' # 12 UTC on Mondays for weekly release; images for GEM and Mimir should have already been pushed
+
+  workflow_dispatch: # for manual testing
+
+jobs:
+  weekly-release-pr:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: imjasonh/setup-crane@v0.1
+
+      - name: Update/regenerate files
+        id: update
+        run: bash .github/workflows/scripts/helm-weekly-release.sh
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v4
+        with:
+          token: ${{ secrets.GH_BOT_ACCESS_TOKEN }}
+          title: Release mimir-distributed Helm chart ${{ steps.update.outputs.new_chart_version }}
```

Comment:
does this skip PR creation if the PR already exists with the same name? open or closed

Reply:
I will discover that

Reply:
if there's a closed PR it opens a new one. It did that quite a few times while I was testing. If the PR is open, it force pushes - see https://github.com/grafana/mimir/pull/2622